riscv-runtime/src/hash.rs

```diff
@@ -27,3 +27,85 @@ pub fn poseidon_gl(data: [u64; 12]) -> [u64; 4] {
 
     poseidon_gl_unsafe(data)
 }
+
+/// Calls the keccakf machine
```

Comment:
minor: function comments here could use periods at the end of sentences

Reply:
Done.

Reply:
small comment but this sentence is still missing the ending period

Reply:
Done